.eslintignore

```diff
@@ -6,3 +6,4 @@ icons/
 dependent-usage-analyzer/
 build-scss.js
 component-generator/templates/test.jsx
+example/
```

Comment:
[curious] If we're adding `example` to .eslintignore, is the NPM script `example-lint` necessary?

Reply:
I think we can remove. I added to have opportunity to run `eslint` from root but I see it is not necessary. The origin intention was to add `eslint` for root directory but then I should also add `npm install` for the `example`. This takes some additional time so I decided it as not good idea. Tests will last longer although `example` is not what should be tested every time.